docs/src/faqs/setup_packaging.rst

```diff
@@ -0,0 +1,108 @@
+What do I need to know about using ``setup.py`` to package dependencies in Klio?
+================================================================================
+
+Using ``setup.py`` or a ``requirements.txt`` file to specify dependencies to install on workers was the original means by which a Python Beam job developer could customize the environment in which their job ran. You can read more about it in the `here <https://beam.apache.org/documentation/sdks/python-pipeline-dependencies/>`_.
+
+To configure a Klio job to run using ``setup.py`` packaging, the user needs to provide the required files (see below) and configure the job to use them by filling in the ``pipeline_options.setup_file`` field in their ``klio-job.yaml`` file:
+
+
+.. code-block:: yaml
+
+    job_name: my-job
+    pipeline_options:
+        setup_file: setup.py
+        # <-- snip -->
+
+Note that this method of setup is mutually exclusive with using the FnAPI for packaging.
```

Comment:
Good opportunity to link to the other FAQ on how to migrate from setup.py to fnapi.